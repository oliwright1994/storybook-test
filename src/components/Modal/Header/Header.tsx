import { ModalHeader as ChakraModalHeader } from '@chakra-ui/core'
import React, { FC } from 'react'

const Header: FC = ({ children }) => {
  return <ChakraModalHeader>{children}</ChakraModalHeader>
}

export default Header
