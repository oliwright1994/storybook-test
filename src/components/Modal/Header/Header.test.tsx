import React from 'react'
import { renderWithBrightTheme } from 'testing/helpers/themeHelper'

import Header from '.'

describe('<Header />', () => {
  const defaultText = 'Test Header'
  it('should render as expected', () => {
    const { container } = renderWithBrightTheme(<Header>{defaultText}</Header>)

    expect(container).toMatchSnapshot()
  })

  it('should render passed children', () => {
    const { getByText } = renderWithBrightTheme(<Header>{defaultText}</Header>)

    const headerEl = getByText(/test header/i)
    expect(headerEl).toBeTruthy()
    expect(headerEl.nodeName).toMatch(/header/i)
  })
})
