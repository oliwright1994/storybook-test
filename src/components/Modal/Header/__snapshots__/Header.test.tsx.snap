// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render as expected 1`] = `
<div>
  <header
    class="css-w7idnj"
  >
    Test Header
  </header>
</div>
`;
